visualization/app/codeCharta/util/treeMapHelper.ts

```diff
@@ -18,6 +19,15 @@ export class TreeMapHelper {
 		return count
 	}
 
+	public static arrayToMap(highlightArray: CodeMapBuilding[]) {
```

Comment:
change signature to `public static buildingArrayToMap(highlightedBuildings: CodeMapBuilding[])` since it's not generic

Reply:
also write a test